### DISCUSSION TITLE
Make fails / hangs on Raspi Zero W

### DISCUSSION THREAD

**Discussion by marcuskuenzel1971** [akuker/RASCSI#1100] [General]
Hello,

I loaded the files from github with all things in the manual and run easyinstall.sh. It starts fine but after a while it seems to hang. The led on the pi is just blinking, even after 30 minutes and more. What could be the reason ?

Greetings

**Comment by rdmark** [comment]
On a Pi Zero the latest piscsi code takes about 135 min to compile. Compiling C++ is a very CPU intensive operation for the poor RPi. You just have to wait a bit longer!

To save time you can cross compile on a faster system, or use the read-built RPi images that we distribute with each release.

**Reply by marcuskuenzel1971** [reply]
sadly it doesn't seems to work. Either it interrupts after 1 or 2 hours with a message from the make process or even after 5 hours and more it hangs an nothing other than a blinking led happens.

Is it possible to copy an piscsi installation from a premade image to my current working pi zero W system ? 

Greetings and thanks,

Marcus

**Reply by rdmark** [reply]
Can you please share the log leading up to and including the error message from make?

Also, which version of the piscsi software are you using, and do you pass any parameters to the easyinstall script?

What other applications are running on your RPi? Do you have an entire desktop environment, or other server daemons running? What you describe sounds like the system running out of memory when compiling. I would try to shut down any unnecessary services, and then have `top` running while the compilation is ongoing to monitor memory usage.

To answer your question: Yes it's theoretically possible to copy the compiled binary files and use them on your RPi as long as they're compiled for the right architecture and linked in the right way. We don't have any facilities or instructions for doing this, however.

**Comment by marcuskuenzel1971** [comment]
unfortunatly it didn't work on my installation but with a premade image it runs well
